core_lib_generator/config_collectors/database.py

```diff
@@ -85,9 +85,14 @@ def _generate_db_config(
     migrate: bool = False
 ) -> dict:
     config = _build_url(db_type, db_name, db_username, db_password, db_port, db_host)
+    if db_type:
+        db_type: str = 'core_lib.connection'
+        if migrate:
+            db_type += '.sql_alchemy_connection_registry.SqlAlchemyConnectionRegistry'

```

Comment:
if the migrate is `False` the db_type string will just be `'core_lib.connection'`, talk with me about it